examples/tutorials/analysis-1d/spectrum_simulation.py

```diff
@@ -164,9 +164,8 @@
 #
 # To do an on off spectral analysis, which is the usual science case, the
 # standard would be to use `SpectrumDatasetOnOff`, which uses the
-# acceptance to fake off-counts. Please also refer to :doc:`simulations in
-# the absence of a background model
-# <spectral_analysis_rad_max.html#dataset-simulations>`
+# acceptance to fake off-counts. Please also refer to `Dataset simulations`
+# of the tutorial :doc:`/tutorials/analysis-1d/spectral_analysis_rad_max`
 # for simulations based on observations of real off counts.
```

Comment:
Please also refer to the `Dataset simulations` section in the :doc:`/tutorials/analysis-1d/spectral_analysis_rad_max` tutorial